.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
What are you expecting this to be used for ?
I think documentations issues could go into other category (bug report or feature request).
Should we improve the other template documentation instead ?

Reply:
It could be used for the following documentation problems:
* Vague; not clear; Incomplete; Misleading; Outdated API documentation
* Any spelling mistake
* Any update in dataset description, `home_url`, `citation`, etc.

In my opinion, the above problems could be labeled as `documentation` issue. This will also help us to filter the real bugs.
Or as you suggested we could add a new `other` issue template.

Reply:
Ok sounds good.